scraper.rb

```diff
@@ -115,6 +115,37 @@ def name_with_title
   SearchPage.new(response: Scraped::Request.new(url: url).response).members
 end.uniq
 
+# The search index and list pages seem to be missing some members of
+# the National Assembly, so add them manually to the pages to be scraped:
+members.concat [
```

Comment:
I thought about that and figured it wasn't a big deal. But I take your point - I've made the chnage you suggest in 271c031.